### PR TITLE
Add extra textfield examples

### PR DESCRIPTION
**Pull request by Topener**

### CODE REVIEW

File: app/controllers/controls/textfield.js [hunk 1]
```diff
@@ -7,14 +7,23 @@ var log = require('log');
...
+    log.args(e.source.id + ' focussed!');
 }
 
 function textFieldBlurred(e) {
-    log.args('Ti.UI.TextField blurred!');
+    focussedTextfield = null;
+    log.args(e.source.id + ' blurred!');
+}
+
+function blurTextfield(){
```
Comment (hansemannn) [inline app/controllers/controls/textfield.js]: ;)

File: app/styles/controls/textfield.tss [hunk 2]
```diff
@@ -1,7 +1,70 @@
...
+        fontSize: 14,
+        fontWeight: 'bold'
+    },
+    width: Ti.UI.SIZE,
+    textAlign: Ti.UI.TEXT_ALIGNMENT_CENTER,
+    top: 20,
+    width: 300
+}
+
+/* Different textfields based on different keyboard types. There are more
```
Comment (hansemannn) [inline app/styles/controls/textfield.tss]: Can you leave the initial `/*` line empty? Just to improve the reading

File: app/views/controls/textfield.xml [hunk 3]
```diff
@@ -1,5 +1,18 @@
...
+            <TextField class="textfield" id="regularTextfield" onChange="textFieldValueChanged" onFocus="textFieldFocussed" onBlur="textFieldBlurred" />
+            <TextField class="textfield" id="phoneTextfield" onChange="textFieldValueChanged" onFocus="textFieldFocussed" onBlur="textFieldBlurred" />
+            <TextField class="textfield" id="emailTextfield" onChange="textFieldValueChanged" onFocus="textFieldFocussed" onBlur="textFieldBlurred" />
+            <TextField class="textfield" id="passwordTextfield" onChange="textFieldValueChanged" onFocus="textFieldFocussed" onBlur="textFieldBlurred" />
+            
+            <Label class="textLabel" platform="ios">
+                You can drag down the keyboard when its visible and the view is scrollable. This is thanks to the keyboardDismissMode property on the wrapping ScrollView
+            </Label>
+            
+        </ScrollView>
```
Comment (hansemannn) [inline app/views/controls/textfield.xml]: Open for discussion here, but I usually prefer not to use too many empty lines (only the ones in L#6 and L#11).

Comment (Topener) [reply app/views/controls/textfield.xml]: I removed them for now... I do find it more clear though.

File: app/controllers/controls/textfield.js (hunk 1 above)
Comment (Topener) [reply app/controllers/controls/textfield.js]: oops! Fixed 👍

**Pull request merged**